docs/sources/alerting/configure-notifications/create-notification-policy.md

```diff
@@ -55,38 +55,33 @@ For more information on notification policies, refer to [fundamentals of Notific
-8. Optionally, enable **Override grouping** to specify the same grouping as the default policy. If this option is not enabled, the default policy grouping is used.
-9. Optionally, enable **Override general timings** to override the timing options configured in the group notification policy.
-10. Click **Save policy** to save your changes.
-
-## Add nested policy
+Child policies are always added under the default policy and are automatically added to the end of the tree structure. If you want to choose where to position your policy, see the section on **Add a sibling policy**.
 
 1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
 1. Click **Notification policies**.
-1. Expand the specific policy you want to update.
-1. Click **+ Add nested policy**, then add the details using information in [Add new specific policy](#add-new-nested-policy).
+1. From the **Choose Alertmanager** dropdown, select an Alertmanager. By default, the **Grafana Alertmanager** is selected.
+1. Click **+New child policy** from the default policy.
+1. In the Matching labels section, add one or more rules for matching alert labels.
+1. In the **Contact point** dropdown, select the contact point to send notification to if alert matches only this specific policy and not any of the child policies.
+1. Optionally, enable **Continue matching subsequent sibling nodes** to continue matching sibling policies even after the alert matched the current policy. When this option is enabled, you can get more than one notification for one alert.
+1. Optionally, enable **Override grouping** to specify the same grouping as the default policy. If this option is not enabled, the default policy grouping is used.
+1. Optionally, enable **Override general timings** to override the timing options configured in the group notification policy.
 1. Click **Save policy** to save your changes.
 
-## Edit notification policies
+## Add a sibling policy
 
-1. In the left-side menu, click **Alerts & IRM**, and then **Alerting**.
+1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
 1. Click **Notification policies**.
-1. Find the policy you want to edit, then click **...** -> **Edit**.
-1. Make any changes using instructions in [Add new specific policy](#add-new-nested-policy).
-1. Save your changes.
+1. Find the child policy you want to create a sibling for.
+1. Click **Add new policy** -> **New sibling above** or **New sibling below**.
+
+   Notification policies are evaluated from top to bottom, so it is key to be able to choose which notification policy receives alerts first. Adding sibling notification policies instead of always inserting a child policy as well as being able tp choose where to insert is key to managing where your alerts go.
```

Comment:
typo

```suggestion
   Notification policies are evaluated from top to bottom, so it is key to be able to choose which notification policy receives alerts first. Adding sibling notification policies instead of always inserting a child policy as well as being able to choose where to insert is key to managing where your alerts go.
```